src/engagement_db_to_analysis/code_imputation_functions.py

```diff
@@ -681,34 +681,7 @@ def _impute_nic_demogs(user, column_traced_data_iterable, analysis_dataset_confi
              f"traced data items")
 
 
-def _get_consent_withdrawn_participant_uuids(column_traced_data_iterable, analysis_dataset_configs):
```

Comment:
https://github.com/AfricasVoices/Engagement-Data-Pipeline/pull/279#discussion_r1362471839
I've moved the function to obtain consent withdrawn in advance before performing any filter because imputing codes after filtering messages, we might miss some stop codes.